templates/ingress.yaml

```diff
@@ -363,7 +363,21 @@ spec:
       name: {{ include "coder.serviceName" . }}
       port:
         number: 8080
-  {{- if not .Values.coderd.replica.enable }}
+  {{- if .Values.coderd.replica.enable }}
```

Comment:
I'm not clear what this change means, exactly -- it seems like the case before would've defaulted to coder/8080? How come the explicit rule is needed?

Reply:
the automatic tls hook with letsencrypt doesn't work unless you specify a rule it seems

Reply:
Ah, TIL! thanks for the info